mantle/build

```diff
@@ -29,7 +29,7 @@ host_build() {
     go build \
         -ldflags "${ldflags}" \
         -mod vendor \
-        -o "bin/$cmd" \
+        -o "../bin/$cmd" \
```

Comment:
Mantle has a separate `.gitignore` that can probably be pruned a bit.

Reply:
Indeed. I'm not sure we need any of the ones in there actually. The toplevel `.gitignore` also could use a cleaning, so I'll keep this for a follow-up.